luigi/tools/range.py

```diff
@@ -101,6 +101,9 @@ class RangeBase(luigi.WrapperTask):
     now = luigi.IntParameter(
         default=None,
         description="set to override current time. In seconds since epoch")
+    param_name = luigi.Parameter(
+        default=None,
```

Comment:
Oh, `positional` is a flag to `luigi.Parameter`.  I didn't know about that.  I'll fix that now.